README.md

```diff
@@ -65,6 +67,8 @@ section-name/02-next-topic-workbook.md
 section-name/0n-last-topic-article.md
 section-name/0n-last-topic-script.md
 section-name/0n-last-topic-workbook.md
+. . .
+workbook/01-section.asciidoc // Contains the part of the workbook matching the section's name.
```

Comment:
Match naming used in the folder structure (above).

```suggestion
workbook/0n-section-name.asciidoc // Contains the part of the workbook matching the section's name.
```

Reply:
Thanks!